.github/workflows/tests.yml

```diff
@@ -3,10 +3,10 @@ name: Build sdist and wheel for testing purposes
 on:
   push:
     branches:
-      - rtmidi-5.0.0
+      - improve-build
   pull_request:
     branches:
-      - rtmidi-5.0.0
+      - improve-build
```

Comment:
Something doesn't seem right here. Does this belong in the master branch?

Reply:
I used this file to test workflow changes. I guess we can delete it now.